### ISSUE TITLE
OptogeneticStimulusSite description

### ISSUE THREAD

**Issue by bendichter** [NeurodataWithoutBorders/nwb-schema#343]
OptogeneticStimulusSite.description is a dataset. This isn't inherently a problem, but it is inconsistent with TimeSeries.description, which is an attribute. 

https://github.com/NeurodataWithoutBorders/nwb-schema/blob/99fbbdc68984ccb7c2fdb30e921f55f152e5b4a2/core/nwb.ogen.yaml#L24-L31

Maybe not worth fixing for 2.x if it will cause compatibility issues, but worth correcting in 3.0

**Comment by oruebel** [comment]
I agree, since this is just a minor inconsistency between the types, we should probably wait with this for 3.0. Thanks for creating the issue. It is helpful to keep track of these issues as we find them.
